src/components/shared/Radio/Radio.tsx

```diff
@@ -7,7 +7,8 @@ export default function Radio({ children, id, nomGroupe, onChange }: Props): Rea
         <input
           id={id}
           name={nomGroupe}
-          onChange={onChange}
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...onChange ? { onChange } : {}}
```

Comment:
est ce que comme ça il y a aussi besoin du `// eslint-disable-next-line react/jsx-props-no-spreading` ❓ 
```suggestion
          ...onChange && { onChange }
```